src/ui/public/notify/notifier.js

```diff
@@ -229,7 +257,7 @@ Notifier.prototype.error = function (err, cb) {
     content: formatMsg(err, this.from),
     icon: 'warning',
     title: 'Error',
-    lifetime: Infinity,
+    lifetime: 300000,
```

Comment:
Ideally this would be configurable, try searching the repo for "courier:maxSegmentCount" to see how it is implemented.